e3sm_to_cmip/cmor_handlers/mpas_vars/fsitherm.py

```diff
@@ -60,7 +62,7 @@ def handle(infiles, tables, user_input_path, **kwargs):
 
     ds = mpas.remap(ds, 'mpasocean', mappingFileName)
 
-    mpas.setup_cmor(VAR_NAME, tables, user_input_path, component='ocean')
+    util.setup_cmor(VAR_NAME, tables, TABLE, user_input_path)
```

Comment:
I feel that the syntax:`(var_name=VAR_NAME, table_path=tables, table_name=TABLE, user_input_path=user_input_path)` is easier to follow than the simplified version`(VAR_NAME, tables, TABLE, user_input_path)`, especially to differentiate `tables` and `TABLE`.

Reply:
I can change it across the board using "sed".  The fact that it exists in 41 modules is clear evidence that this is NOT where we should be calling "cmor.setup".  The only variables in the call parameters are VAR_NAME and TABLE, and in principle we would not even know which handler module to select if we did not already have (essentially) this information BEFORE we call the handler.

Someday ... we should consolidate this processing.  I know we want to "precompile" and not "interpret" for performance reasons, buy that should be solvable with an array of pre-compiled formulas indexed by variable and table., whose inputs are pointers to locations where data will be written for access.

In the meantime, I'll make the changes and push.

Reply:
Converted to named parameters for mpas handler calls to util.setup_cmor.

Reply:
@chengzhuzhang  Also converted 4 residual "vars" handlers (areacella, clisccp, orog, sftlf), and had to change util because the named parameter was "varname", not "var_name".  Now it is and all should be consistent.